pkg/kubelet/apis/config/validation/validation_others.go

```diff
@@ -31,5 +31,9 @@ func validateKubeletOSConfiguration(kc *kubeletconfig.KubeletConfiguration) erro
 		return fmt.Errorf("invalid configuration: singleProcessOOMKill is only supported on linux")
 	}
 
+	if kc.UserNamespaces != nil && kc.UserNamespaces.IDsPerPod != nil {
+		return fmt.Errorf("invalid configuration: userNamespaces.idsPerPod is only supported on linux")
```

Comment:
if the entire userNamespaces feature is unsupported, prevent use of the whole config stanza?

```suggestion
	if kc.UserNamespaces != nil {
		return fmt.Errorf("invalid configuration: userNamespaces is only supported on linux")
```

Reply:
done